modules/user_accounts/php/edit_user.class.inc

```diff
@@ -254,8 +254,8 @@ class Edit_User extends \NDB_Form
         unset($values['NA_UserID']);
         // generate new password
         if (!empty($values['NA_Password'])) {
-            $values['Password_hash']   = \Utility::randomString();
-            $values['Password_expiry'] = '1990-04-01';
+            $values['Password_hash']    = \Utility::randomString();
+            $values['Password_expired'] = true;
```

Comment:
```suggestion
            $values['Password_expired'] = 1;
```
Although 1 can be compared to true,  tinyint is a number type. 
MySQL 5.7 accept true and false as 1 and 0 for tinyint but it might not be always the case. Safer to use 0 and 1.

Reply:
True and false are semantically correct. MySQL converts "BOOLEAN" datatypes to `TINYINT(1)`